packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -1,12 +1,60 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    linkActiveFontColor,
+    linkFontColor,
+    smallPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        overflow: auto;
+        block-size: 100%;
+        min-block-size: 36px;
+    }
+
+    .viewer {
+        font: inherit;
+        outline: none;
+        box-sizing: border-box;
+        position: relative;
+        color: inherit;
+        min-inline-size: 100px;
+        padding: ${smallPadding};
+        margin-block-end: 10px;
```

Comment:
Rick mentioned he was sending over some preliminary visual designs this week. Are they incorporated in this styling code? Or is there a follow-up task to adopt them?

One thing that stands out to me are these `10px` margins and `100px` sizes. Almost everything in Nimble is done on a `4px` grid so I'm worried they're not correct. If there are plans to improve later then I'm ok to leave the styling as-is for now, just want to make sure it's tracked.